spec/rails_helper.rb

```diff
@@ -28,6 +34,24 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
+
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app,
+                                      js_errors: false,
+                                      phantomjs_options: ['--proxy-type=none'],
+                                      timeout: 180,
+                                      :phantomjs_logger => File.open("#{Rails.root}/log/test_phantomjs.log", "a"),
+                                      extensions: [])
+  end
+
+  Capybara.javascript_driver = :poltergeist
+  Capybara.default_selector = :css
+  Capybara.default_wait_time = 5
+
+  Capybara::Screenshot.register_filename_prefix_formatter(:rspec) do |example|
+    "screenshot-#{example.description.gsub(' ', '-').gsub(',', '').gsub('/', '-').gsub('"','')}"
```

Comment:
Line is too long. [96/80]<br>Use `delete` instead of `gsub`.<br>Use `tr` instead of `gsub`.<br>Space missing after comma.